app-admin/logstash-bin/logstash-bin-5.6.13.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Can 7 be used in here? (Same applies to 2nd version bump)

Reply:
Unfortunately, the inherited java-utils-2 eclasses uses versionator, which is banned in EAPI=7.

Reply:
Yep, saw it got inherited and had a feeling there might be something. Thanks for checking!

Reply:
Thanks for the review :)